gravity.sh

```diff
@@ -391,6 +394,7 @@ gravity_ParseFileIntoDomains() {
 
       # Remove exceptions
       comm -23 "${destination}" <(sort "${destination}.exceptionsFile.tmp") > "${source}"
+	  chmod 644 "${source}"
```

Comment:
The `>` operator does not modify the file permissions if the file is already created. `source` is already created, so we do not need to reset the permissions.

If this is not the case, then there is one more issue. Please use spaces instead of tabs.